src/components/course/styles/CourseRunSelector.scss

```diff
@@ -0,0 +1,9 @@
+.btn {
+  &.save-selection {
```

Comment:
curious why we need custom styles for the "go" button?

what happens when the button text changes to something longer than "go" (e.g., if the page was i18n-ready and the button said "vamos" instead)? would the width of 44px be too short?

Reply:
Great catch, yeah looks horrible for anything longer than "go". Full disclosure I just stole this styling from Prospectus ☹️ 

How do we feel about something like this, without an explicit width it looks like a sad little circle even with padding.
```
.btn {
  &.save-selection {
    padding: .25em;
    min-width: 3em;
    line-height: 1;
  }
}

```
<img width="367" alt="Screen Shot 2020-06-17 at 11 50 55 AM" src="https://user-images.githubusercontent.com/1559107/84920183-d4c9de00-b090-11ea-91d5-2f358132e853.png">
<img width="274" alt="Screen Shot 2020-06-17 at 11 50 38 AM" src="https://user-images.githubusercontent.com/1559107/84920186-d5627480-b090-11ea-86e5-375a0abcb09e.png">

Reply:
Could we stick to using Bootstrap classes instead of adding custom styles here? example: `className="py-0 px-1"`